paper.md

```diff
@@ -0,0 +1,400 @@
+Several of the Boost libraries have been already incorporated into the C++ standard, and we build off of those with Empirical.
+However, Boost does not contain libraries for web-based GUI tools, configuration management, or data management specifically tailored to scientific software.
+
+### Emscripten
+
+Emscripten is available at <https://emscripten.org/> [@zakai2011emscripten].
+It provides cross-compilation from C++ to WebAssembly and we use it in Empirical.
+Empirical's tools build abstractions from Emscripten intrinsics tailored to visualization and interactive control of scientific simulations.
+
+
+### Cheerp
+
+Cheerp, another C++ to WebAssembly compiler, is available at <https://leaningtech.com/cheerp/>.
+Like Emscripten, Cheerp provides primariliy low-level APIs for interaction with browser GUI elements.
+
+### Non-C++ Comparable Software
+
+<!-- Not going to include discussion, just citations -->
+
+<!--
+these are more agent-based simulations which aren't the focus of this paper
+* FlameGPU [@richmond2010high]
+* NetLogo [@tisue2004netlogo]
+* Cell Collective [@helikar2012cell]
+ -->
+
+* [TinyGo](https://tinygo.org/) <!-- in-browser web interface compiler for go -->
+* [WebIO](https://juliagizmos.github.io/WebIO.jl/latest/) <!-- in-browser web interface library for julia -->
+* [GWT](http://www.gwtproject.org/)  <!-- in-browser web interface compiler for java -->
+* [yew](https://yew.rs/) <!-- in-browser web interface library for rust -->
+* Pyodide [@developers2021pyodide] <!-- web assembly python interpreter -->
+* Shiny [@chang2020shiny] <!-- web visualizations that interface with R code running on a server -->
+
+
+## Projects Using the Software
+
+* [Aagos](https://lalejini.com/Aagos/web/Aagos.html) [@gillespie2018changing]
+    * An interactive model for empirically studying how evinronmental change affects the evolution of genetic architectures.
+* [conduit](https://conduit.fyi) [@moreno2020conduit]
+  * A C++ library that wraps intra-thread, inter-thread, and inter-process communication in a uniform, modular, object-oriented interface, with a focus on asynchronous high-performance computing applications.
+* [Dishtiny](https://mmore500.com/dishtiny) [@moreno2019toward]
+  * An interactive web demo of an artificial life platform built to study major transitions in evolution.
+* [ecology in evolutionary computation explorer](https://emilydolson.github.io/ecology_of_evolutionary_computation/web/interaction_networks.html) [@dolson2018evological]
+  * Interactive exploration of interaction networks in evolutionary computation under different selection schemes.
+* [Symbulation](https://anyaevostinar.github.io/SymbulationEmp/web/symbulation.html) [@vostinar2017suicide]
+  * An interactive artificial life model focused on the evolution of symbiosis between parasitism to mutualism.
+    Also features a game to recreate major results from published work.
```

Comment:
Game no longer in place, can remove this sentence :)

Reply:
```suggestion
```